### PR TITLE
Include the DicomImagePrintTags.py output in Sphinx docs

### PR DESCRIPTION
**Pull request by mbopfNIH**
Added sphinx_exec_code directive and used it to run the Python example as a subprocess in the rst file using the Image0075.dcm as input.

Fixes  issue #1798.

### CODE REVIEW

File: Examples/DicomImagePrintTags/Documentation.rst [hunk 1]
```diff
@@ -32,3 +32,17 @@ Code
...
+Sample Output
+-------------
+
+Running the above Python script with the image  "docs/images/Image0075.dcm" produces the following output:
+
+.. exec_code::
+
+    # --- hide: start ---
+    import subprocess
+    result = subprocess.call(['python',
```
Comment (blowekamp) [inline Examples/DicomImagePrintTags/Documentation.rst:45]: Please see this change for a better way to get the current python executable.
64bd1f84

**Comment by blowekamp** [comment]
Looks pretty good.

**Comment by mbopfNIH** [comment]
I've changed the code to eliminate the trailing whitespace, and improve the method of determining the python executable. But I don't know how to get the ghostflow check to re-run with my new commit. Also, is there a way to stop the old tests and restart with the new code?

**Comment by mbopfNIH** [comment]
Made suggested changes to get correct python executable, fixed ghostflow issue, and squashed commits. Build appears to be failing only because of that numpy read-only issue. (At least I think so).

File: docs/requirements.txt [hunk 2]
```diff
@@ -2,3 +2,4 @@ sphinx-tabs
 cmake
 sphinx_rtd_theme
 sphinx~=5.3.0
+sphinx_exec_code~=0.8
```
Comment (blowekamp) [inline docs/requirements.txt:5]: SimpleITK is also going to be needed here.

Comment (mbopfNIH) [reply docs/requirements.txt:5]: Right. Should I specify a version? Maybe >=2.2 ?

Comment (blowekamp) [reply docs/requirements.txt:5]: How about "~=2.2.0"

Comment (mbopfNIH) [reply docs/requirements.txt:5]: Okay, but that wouldn't match 2.3 and above, right? So would that require an update this requirements.txt each time we push a minor upgrade?

Comment (blowekamp) [reply docs/requirements.txt:5]: Yes. But it would also ensure if an incompatible change was made the older versions could still be built.

**Comment by mbopfNIH** [comment]
I may have done an unnecessary merge to get my branch synced with the upstream master. But the code change is simply the extra simpleitk~=2.2.0 in requirements.txt.

**Comment by blowekamp** [comment]
The way to update a topic or PR branch with the changes of master is to rebase it onto the master branch. We don't want the commit history to have unnecessary merges in the history which would create confusion and clutter. Also, the changes from master were not needed for this topic/PR.

**Pull request closed without merging**